frontend/test/playwright/playwright.config.ts

```diff
@@ -1,14 +1,5 @@
-import path from "path"
-
-import { addAliases } from "module-alias"
-
 import type { PlaywrightTestConfig } from "@playwright/test"
 
-addAliases({
-  "~": path.resolve(process.cwd(), "src"),
-  "~~": process.cwd(),
-})
-
```

Comment:
This (and the same in the storybook playwright config) is no longer needed now that the directory structure of the playwright container matches the real structure, which in turn allows Playwright to use our regular tsconfig's paths configuration to resolve aliases.